server/nuxeo-nxr-server/src/main/resources/templates/common/nuxeo.defaults

```diff
@@ -56,3 +56,5 @@ nuxeo.cache.concurrencylevel=500
 
 # boolean to switch back to former hot reload mechanism
 nuxeo.hotreload.compat.mechanism=false
+
+nuxeo.storage.transient.ttl=4320
```

Comment:
I would prefer `nuxeo.transientstore.ttl` as a property name (we already have another one related to transient stores, `nuxeo.transientstore.provider`).
And therefore it would make sense for S3 to have `nuxeo.s3storage.transientstore.ttl` for example

Reply:
works for me, i drew inspiration from `nuxeo.storage.azure.*` as it seemed nicer to read (would rather have `nuxeo-storage.(transient|s3|azure|...).*` than the current naming..).